autoload/clap.vim

```diff
@@ -85,16 +83,6 @@ let g:clap_multi_selection_warning_silent = get(g:, 'clap_multi_selection_warnin
 
 let g:clap_popup_border = get(g:, 'clap_popup_border', 'rounded')
 
-function! clap#builtin_providers() abort
```

Comment:
Can we extract these `tagfiles` provider unrelated code into another PR?

Reply:
Will do